fe/be-java-extensions/java-common/src/main/java/org/apache/doris/common/jni/vec/TableSchema.java

```diff
@@ -0,0 +1,78 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.common.jni.vec;
+
+import org.apache.doris.thrift.TPrimitiveType;
+
+import com.google.common.annotations.VisibleForTesting;
+
+/**
+ * Used to parse the file structure of table-value-function type.
+ * like avro file.
+ */
+public class TableSchema {
+
+    private final String[] fields;
+    private final TPrimitiveType[] schemaTypes;
```

Comment:
Column type is defined in `org.apache.doris.common.jni.vec.ColumnType`, maybe we should use `ColumnType`, not `TPrimitiveType`. Because `ColumnType` has provide the `parseType` method, and get type is an inverse method.